src/lavinmq/shovel/shovel.cr

```diff
@@ -60,6 +61,10 @@ module LavinMQ
         if @queue.nil? && @exchange.nil?
           raise ArgumentError.new("Shovel source requires a queue or an exchange")
         end
+        @args = AMQ::Protocol::Table.new
+        @consumer_args.try &.each do |k, v|
+          @args[k] = v
```

Comment:
How will `JSON::Any` be handled from here? Should it be `v.as_s?` or something?

Reply:
I don't `v.as_s?` is needed for it to work, but it's probably best to be specific

Reply:
Oh, `AMQ::Protocol::Table` accepts `JSON::Any` as value. Didn't know that (and I guess it compiled for you 🤦).